### PR TITLE
[15.0][mig] sale_order_line_sequence

### PR DESCRIPTION
**Pull request by rajpatel2240-odoo**

### CODE REVIEW

**Review by lfreeke — APPROVED** [review]
Functional test :+1:

File: sale_order_line_sequence/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+# Copyright 2017 Serpent Consulting Services Pvt. Ltd.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Sale Order Line Sequence",
+    "summary": "Propagates SO line sequence to invoices and stock picking.",
+    "version": "15.0.1.0.0",
+    "author": "Eficent, Serpent CS, Odoo Community Association (OCA)",
```
Comment (MiquelRForgeFlow) [inline sale_order_line_sequence/__manifest__.py:9]:
```suggestion
    "author": "ForgeFlow, Serpent CS, Odoo Community Association (OCA)",
```

File: sale_order_line_sequence/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
```
Comment (MiquelRForgeFlow) [inline sale_order_line_sequence/__manifest__.py:1]:
```suggestion
# Copyright 2017 ForgeFlow S.L.
```
in all files

File: sale_order_line_sequence/readme/CONTRIBUTORS.rst [hunk 3]
```diff
@@ -0,0 +1,3 @@
+* Eficent Business and IT Consulting Services S.L. <contact@eficent.com>
```
Comment (MiquelRForgeFlow) [inline sale_order_line_sequence/readme/CONTRIBUTORS.rst:1]:
```suggestion
* Forgeflow S.L. <contact@forgeflow.com>
```

**Review by MiquelRForgeFlow — CHANGES_REQUESTED** [review]
Please

**Comment by LoisRForgeFlow** [comment]
@rajpatel2240-odoo Could you attend comments to finish the migration?

**Comment by MiquelRForgeFlow** [comment]
Superseded by https://github.com/OCA/sale-workflow/pull/2311.

**Pull request closed without merging**